qa/tasks/mgr/dashboard/test_rbd.py

```diff
@@ -525,7 +531,7 @@ def test_flatten(self):
         self.assertIsNotNone(img['parent'])
 
         self.flatten_image('rbd_iscsi', 'img1_snapf_clone')
-        self.assertStatus(200)
+        self.assertStatus([200, 201])
```

Comment:
The only status code returned by this operation is 200

Reply:
According to https://github.com/ceph/ceph/pull/21066/files#diff-52895dc8193e5a52e6bebbdcb34b9bbdR192 it can also return 201

Reply:
@sebastian-philipp you're right